modules/search/src/endpoints.rs

```diff
@@ -36,3 +40,23 @@ async fn search_advisories(
         service.search_advisories(q, sort, paginated).await?,
     ))
 }
+
+#[utoipa::path(
+    context_path = "/api/v1/search",
+    tag = "search",
+    params(
+        ("q", Query, description = "The query expression"),
```

Comment:
```
params(
    SearchOptions
)
```

will bring all the docs for both `q` and `sort` and simplify the writing of each search-able endpoint.

Similar with `Paginated`.